CMakeLists.txt

```diff
@@ -1,17 +1,26 @@
-# Copyright (c) Open Enclave SDK contributors.
-# Licensed under the MIT License.
+# Copyright (c) Open Enclave SDK contributors. Licensed under the MIT License.
```

Comment:
Is this suggested by cmake format?

Reply:
Yes. The reason was that the .cmake-format.json was missing. I added that as well as formate-cmake to pre-commit. Currently the pre-commit works only on Linux. It think it is fine for now.